.github/workflows/publish-packages.yml

```diff
@@ -3,7 +3,6 @@ name: Publish packages
 on:
   workflow_run:
     workflows: [Bump version]
-    branches: [main]
```

Comment:
This is ok now that `Bump version` will fail unless run against a valid pre-release.